### ISSUE TITLE
padevents pings rewrite

### ISSUE THREAD

**Issue by RheingoldRiver** [TsubakiBotPad/padbot-cogs#120]
TLDR add a config object, also move it out of the `padevents` command group

`^autoeventping add <key>`
`^autoeventping show <key>`
`^autoeventping remove <key>`
`^autoeventping list`

* Searchstring - `^autoeventping set event <key> <searchstring>` - the string to be in the dungeon name
* Searchmode - `^autoeventping set searchmode <key> <mode>` - either `regex` or `plain`, default should be `plain`
* Set three different roles - `^autoeventping set roles <key> <red> <blue> <green>`
* Set one role - `^autoeventping set redrole <key> <red>`, etc
* Enable - `^autoeventrping set enabled <key>`
* Disable - `^autoeventrping set disable <key>`
* Offset - `^autoeventrping set offset <key> <minutes>` - this is the number of minutes before the dungeon starts that the ping should happen. The format of the message should now say, `will start in {} minutes!`

**Comment by RheingoldRiver** [comment]
closed by #126 and #127
